tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,179 @@
+"""
+A test case to verify after deleting pvc whether
+size is returned to backend pool
+"""
+import logging
+import time
+import pytest
+
+from ocs import constants, defaults
+import ocs.exceptions as ex
+from tests import helpers
+from ocsci.testlib import tier1, ManageTest
+from utility import templating
+from resources import pod
+from ocs import ocp
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret
+)
+
+logger = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+PV = ocp.OCP(
+    kind='PersistentVolume', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+def check_ceph_used_space():
+    """
+    Check for the used space in cluster
+    """
+
+    # ToDo: after running IO's or deleting pod & pvc takes some time
+    time.sleep(20)
+    ct_pod = pod.get_ceph_tools_pod()
+    pods = ct_pod.exec_ceph_cmd(ceph_cmd="ceph status")
+    assert pods is not None
+    used = pods['pgmap']['bytes_used']
+    GB = (1024 * 1024 * 1024)
+    used_in_gb = used / GB
+    return used_in_gb
+
+
+def run_io(pod_obj):
+    """
+    Run io on the mount point
+    """
+
+    # Run IO's
+    pod_obj.exec_cmd_on_pod(
+        command="dd if=/dev/urandom of=/var/lib/www/html/dd_a bs=10M count=950"
```

Comment:
@Akarsha-rai @ebenahar @nehaberry @vasukulkarni @keesturam we need to find a way to simulate IOs in a more realistic way.
urandom is time-consuming and acts as the bottleneck in such cases.

Reply:
any suggestions what to use? should I use cirros-pod?

Reply:
@RazTamir in the long run, we would have to refrain from running any IO with "dd" command as was communicated to us in cns-qe. Having said that, even cirros pod uses dd IO. 

Thus, IMO, till we come up with some realistic app pods and IO pattern, maybe we can continue with this approach. 

Some of the approved and realistic IO could be:
1. pgsql
2. mongodb
3. https://github.com/JohnStrunk/fs-performance-container/blob/master/entry.sh 
etc.

Reply:
I raised it here so we'll have an idea/discussion around this but I'm not going to block this PR because of that